src/mats_l1_processing/photometer.py

```diff
@@ -0,0 +1,93 @@
+#%%
+import numpy as np
+from mats_l1_processing.instrument import Photometer
+from pandas import DataFrame
+
+def calibrate_pm(df: DataFrame, photometer: Photometer):
+    
+# ===============================================================================================================
+# Calibration starts here
+# PM1 is Bkg photometer, 754BP3 nm / PM2 is A-band photometer, 763BP8 nm
+#   Change to bits/s
+    PM1_Sig_bit = df['PM1S']/df['PM1SCNTR']
+    PM1_Tpd_bit = df['PM1A']/df['PM1ACNTR']
+    PM1_Tif_bit = df['PM1B']/df['PM1BCNTR']
+    PM2_Sig_bit = df['PM2S']/df['PM2SCNTR']
+    PM2_Tpd_bit = df['PM2A']/df['PM2ACNTR']
+    PM2_Tif_bit = df['PM2B']/df['PM2BCNTR']
+    pm_texp = round(df['PM1SCNTR']/166) # exposure time in seconds, 1 second "integration" is 166-167 sampling cycles
```

Comment:
With the comment this is ok, but generally I would prefer defining a constant `SAMPLING_CYCLES = 166` or similar and using that.

Reply:
done